ecs-agent/acs/session/attach_resource_responder.go

```diff
@@ -202,6 +202,10 @@ func validateAttachmentAndReturnProperties(message *ecsacs.ConfirmAttachmentMess
 		if err != nil {
 			return nil, errors.Wrap(err, "resource attachment validation by attachment type failed")
 		}
+		err = resource.ValidateFileSystemType(attachmentProperties[resource.FileSystemKey])
```

Comment:
nit - can we have some unit tests to enforce/make sure 
- all allowed file system are allowed as expected
- others will be disallowed/failed as expected

Reply:
yep, changes were made to the attach resource responder unit test to check for invalid values for file system types. I'll add some changes for happy cases as well